src/pages/HomeLogin/HomeLogin.jsx

```diff
@@ -1,5 +1,86 @@
+import { useState } from "react";
+import styled from "styled-components";
+import { useNavigate } from "react-router-dom";
+import HomeLoginTitle from "@/components/HomeLogin/HomeLoginTitle";
+import HomeLoginMbti from "@/components/HomeLogin/HomeLoginMbti";
+import HomeLoginBtn from "@/components/common/HomeLoginBtn";
+import mbtiCircles from "@/constants/HomeLogout/mbtiCircles";
+import HomeLoginMbtiContent from "@/components/HomeLogin/HomeLoginMbtiContent";
+import mbtiContent from "@/constants/HomeLogin/mbtiContent";
+import mbtiTitle from "@/constants/HomeLogin/mbtiTitle";
+
 function HomeLogin() {
-  return <div>HomeLogin</div>;
+  const userName = "유승빈";
+  const skinType = "DRPT";
+  const mbtiCircle = mbtiCircles.find((circle) => circle.text === skinType);
+  const textColor = mbtiCircle?.color;
+  const initialMbtiCircle =
+    mbtiCircles.find((circle) => circle.text === skinType) || mbtiCircles[0];
+  const [selectedMbtiCircle, setSelectedMbtiCircle] =
+    useState(initialMbtiCircle);
+  const [mbtiContentText, setMbtiContentText] = useState(mbtiContent[skinType]);
+  const [mbtiContentTitle, setMbtiContentTitle] = useState(mbtiTitle[skinType]);
+  const [selectedIndex, setSelectedIndex] = useState(null);
+
+  const navigate = useNavigate();
+
+  const handleMbtiClick = (circle, index) => {
+    if (selectedIndex === index) {
+      setSelectedMbtiCircle(null);
+      setMbtiContentText("");
+      setMbtiContentTitle("");
+      setSelectedIndex(null);
+    } else {
+      setSelectedMbtiCircle(circle);
+      setMbtiContentText(mbtiContent[circle.text]);
+      setMbtiContentTitle(mbtiTitle[circle.text]);
+      setSelectedIndex(index);
+    }
+  };
```

Comment:
이 부분 삼항 연산자 사용해서 코드 간소화할 수 있을 것 같아요!

```suggestion
  const handleMbtiClick = (circle, index) => {
    const isSelected = selectedIndex === index;
    setSelectedMbtiCircle(isSelected ? null : circle);
    setMbtiContentText(isSelected ? "" : mbtiContent[circle.text]);
    setMbtiContentTitle(isSelected ? "" : mbtiTitle[circle.text]);
    setSelectedIndex(isSelected ? null : index);
  };
```